src/bbcode.cpp

```diff
@@ -181,7 +183,7 @@ class BBCodeMap {
     m_TagMap["email="]  = std::make_pair(QRegExp("\\[email=\"?([^\\]]*)\"?\\](.*)\\[/email\\]"),
                                          "<a href=\"mailto:\\1\">\\2</a>");
     m_TagMap["youtube"] = std::make_pair(QRegExp("\\[youtube\\](.*)\\[/youtube\\]"),
-                                         "<a href=\"http://www.youtube.com/v/\\1\">http://www.youtube.com/v/\\1</a>");
+                                         "<a href=\"https://youtu.be./\\1\">https://youtu.be/\\1</a>");

```

Comment:
Why did you choose the shortlink and not the full link? This adds an extra redirect.

Reply:
No reason, I simply copied the links format that I found on nexus. Should I change it to 
`https://www.youtube.com/watch?v=` ?

Reply:
I think the long url would be better.

Reply:
Done 👍